__index.md

```diff
@@ -0,0 +1,137 @@
+---
```

Comment:
Seems like some sort of merge conflict/error going on here.

I guess this is supposed to be an edit to https://github.com/apache/jena-site/blob/main/source/documentation/tools/__index.md but it somehow ended up at the top level of the repository instead

Reply:
The new text starts at:
`### Command Line Tools Quick Reference`
I diff'ed against the exitsing tools/__index file and nothing before that line has changed.

Reply:
Sorry, I see that I appended the new material to the wrong __index.md file. I just moved it to the documentation/tools/ one and pushed it to https://github.com/bobdc/jena-site/tree/JENA-2110. Does it look OK now?